test/parallel/test-util-emit-experimental-warning.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+// Flags: --expose-internals
+const common = require('../common');
+const assert = require('assert');
+
+// This test ensures that the emitExperimentalWarning in internal/util emits a
+// warning when passed a unsupported feature and that it simply returns when
+// passed a supported feature.
```

Comment:
The description isn't correct. The second part "and that it simply returns when passed a supported feature." should say "and that it simply returns when passed the same feature multiple times."

Can this also be moved after the require below.

Reply:
@apapirovski : Thanks for the feedback. I've updated the PR. Kindly review now.